syntax = "proto3";
package StreamMessage;

message MsgProto {
  string subject     = 2;  // subject
  bytes  data        = 3;  // payload
  int64  timestamp   = 4;  // received timestamp
  string customer_id = 5; // customer id to which this data belongs
  string msp_id      = 6; // optional field indicating the msp_id 
}